### PR TITLE
fixed border issue

### PR DESCRIPTION
**Pull request by MeenuKumari28**
fixed border issue

### CODE REVIEW

File: layout/container.ftd [hunk 1]
```diff
@@ -13,10 +13,10 @@ optional ftd.resizing height:
...
 ds.gap margin-horizontal: $ds.spaces.vertical-gap.small
 integer border: $ds.borders.uniform.none
-integer border-top: $ds.borders.none
-integer border-bottom: $ds.borders.none
-integer border-left: $ds.borders.none
-integer border-right: $ds.borders.none
+optional integer border-top: $ds.borders.none
+optional integer border-bottom: $ds.borders.none
+optional integer border-left: $ds.borders.none
+optional integer border-right: $ds.borders.none
```
Comment (amitu) [inline layout/container.ftd:19]: If you specify a default value, you do not need optional!

@Arpita-Jaiswal / @harshdoesdev lets add a lint / warning for this (and lets add --strict mode soon, so our code does not contain this mistake).

File: borders.ftd [hunk 2]
```diff
@@ -1,11 +1,10 @@
 -- record border-data:
-integer none:
+optional integer none:
```
Comment (amitu) [inline borders.ftd:2]: Why?

File: layout/container.ftd (hunk 1 above)
Comment (amitu) [reply layout/container.ftd:19]: @MeenuKumari28, remove the part after the `:` when you are making something optional.

File: borders.ftd (hunk 2 above)
Comment (amitu) [reply borders.ftd:2]: This is not needed, you added it because you made optional and still provided a default value.

Comment (MeenuKumari28) [reply borders.ftd:2]: @amitu This is because the `none` value is overwriting the uniform value.

**Pull request merged**